### DISCUSSION TITLE
HTTP add-on is looking for contributors by end of November

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/http-add-on#523] [Announcements]
**Update (22/11/2022) - Locking discussion as we are kicking-off new collaboration and closed the survey. Everyone is welcome to contribute though, but this process is closed!**

On Nov 25, 2020, we started the HTTP add-on based on @arschles his initial POC which closed a big gap in KEDA's story - HTTP autoscaling without a dependency on an external system, such as Prometheus.

To this day, the autoscaling community has a very high demand for a solution in this area that auto scales and works in the same manner as the KEDA core.

With the add-on, we want to cover all traffic patterns ranging from ingress, to service meshes and service-to-service communication and make it super simple to autoscale (and with scale-to-zero support).

As time passed by, we've seen @arschles lead the HTTP add-on to a beta stage and have defined the [requirements for it to become a GA add-on](https://github.com/orgs/kedacore/projects/6/views/9).

Unfortunately we have seen very few contributions to the project and @arschles has switched roles and is unable to maintain it anymore.

📢 **Because of that, we are doing an active call for contributors and will assess the future of the HTTP add-on by the end of November based on the community and (hopefully) contribution growth. Until then, the project is on-hold.** 📢

**If we do not have at least 3 contributors take charge we will have to see what alternatives we have.**

We'd also want to take the time to say a big thank you to @arschles for his hard work on this and hopefully we can continue taking it forward!

**Comment by 26tanishabanik** [comment]
@tomkerkhove , I would like to work on this issue

**Reply by tomkerkhove** [reply]
Awesome, we'll get in touch soon!

**Comment by neelanjan00** [comment]
Hi @tomkerkhove, I would like to work on the requisite action items as well.

**Reply by tomkerkhove** [reply]
Awesome, we'll get in touch soon!

**Comment by prashanth-volvocars** [comment]
Hey,

i would love to contribute as well

**Reply by tomkerkhove** [reply]
Awesome, we'll get in touch soon!

**Comment by gh4x** [comment]
I am also open to helping. This scale to zero functionality is great.

**Reply by tomkerkhove** [reply]
Awesome, we'll get in touch soon!

**Comment by Shulammite-Aso** [comment]
Hi @tomkerkhove I would love to become a contributor too.

**Reply by tomkerkhove** [reply]
Awesome, we'll get in touch soon!

**Comment by tomkerkhove** [comment]
@Shulammite-Aso @gh4x @prashanth-volvocars  @neelanjan00  @26tanishabanik Great to see the reaction from the community! As a next step, would you mind filling in this Google Form please?

https://forms.gle/bMCvhtf7aq4WQoDz8

**Reply by 26tanishabanik** [reply]
> https://forms.gle/bMCvhtf7aq4WQoDz8

Filled the form, @tomkerkhove

**Reply by tomkerkhove** [reply]
Gentle reminder

**Comment by prashanth-volvocars** [comment]
Done with form.

Regards,
Prashanth R

On Mon, 24 Oct 2022 at 11:55 AM, Tom Kerkhove ***@***.***>
wrote:

> Gentle reminder
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kedacore/http-add-on/discussions/523#discussioncomment-3947674>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS7VT4RVO54XIQO7KKE3TETWEYTVRANCNFSM6AAAAAAQWPGBPE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Prashanth R

**Comment by mrinalvirnave** [comment]
@tomkerkhove I would like to contribute as well. I filled out the google form pre-emptively. Hope that's OK.

**Reply by tomkerkhove** [reply]
Certainly, thank you!

**Comment by miguelvr** [comment]
@tomkerkhove I would be happy to help however I can too. 

There is a gap in the K8s ecosystem for a non-intrusive scale to zero solution. I think this project fills that gap perfectly and I would love to help see it succeed!

**Reply by tomkerkhove** [reply]
Awesome, thank you! We are doing a meeting tomorrow - Please fill in [forms.gle/bMCvhtf7aq4WQoDz8](https://forms.gle/bMCvhtf7aq4WQoDz8) and let me know which one of the options you prefer:

- Tuesday Nov 22nd, 8AM UTC
- Tuesday Nov 22nd, 4 PM UTC/8 AM Pacific

**Comment by yxxhero** [comment]
@tomkerkhove Hi. I would like to contribute as well. I have been involved in many open-source projects, including K8S, HELM, HARBOR, etc. I have very relevant experience and hope to get a reply. For more information, please visit my GitHub homepage

**Reply by tomkerkhove** [reply]
Awesome, thank you! We are doing a meeting tomorrow - Please fill in [forms.gle/bMCvhtf7aq4WQoDz8](https://forms.gle/bMCvhtf7aq4WQoDz8) and let me know which one of the options you prefer:

- Tuesday Nov 22nd, 8AM UTC
- Tuesday Nov 22nd, 4 PM UTC/8 AM Pacific

**Reply by yxxhero** [reply]
@tomkerkhove oh. Sorry. my English is not good.

**Reply by tomkerkhove** [reply]
No worries!

**Reply by yxxhero** [reply]
ok.

**Comment by mrinalvirnave** [comment]
@tomkerkhove 
Is there a meeting link that I am missing somewhere?
I plan to attend the 4 PM UTC/8 AM Pacific meeting.

**Reply by tomkerkhove** [reply]
I was waiting for people to reply to my email, feel free to drop me a note on kerkhove.tom@gmail.com

**Comment by tomkerkhove** [comment]
Locking discussion as we are kicking-off new collaboration and closed the survey. Everyone is welcome to contribute though, but this process is closed!
